front/src/modules/trainschedule/components/DriverTrainScheduleV2/DriverTrainScheduleHeaderV2.tsx

```diff
@@ -46,7 +46,8 @@ const DriverTrainScheduleHeaderV2 = ({
     <>
       <div className="d-flex align-items-center">
         <h1 className="text-blue flex-grow-1">{train.train_name}</h1>
-        {isEco(simulatedTrain) && (
+        {(isEco(simulatedTrain) ||
+          (train.schedule && train.schedule.some((item) => item.arrival !== null))) && (
```

Comment:
The second check could be added inside `isEco()` function. Moreover, rather than having to pass 2 parameters (`train` and `simulatedTrain`) to the function, you can only pass `train` and do @flomonster proposal and checking inside if one the schedule arrivals is not null OR if one of the margin values is not 0% and not 0min/100km.

```suggestion
        {(isEco(train) && (
```